### DISCUSSION TITLE
Read-only version - how it's done?

### DISCUSSION THREAD

**Discussion by janswist** [ueberdosis/tiptap#784] [General Questions] [answered]
**Describe the bug**
Not sure if bug, but when I do `editable: false` I still have this focus border around the editor. I'm not sure why in your example it's as it should - is it out of the box or do you use some special styling there?

** Screenshots **
![image](https://user-images.githubusercontent.com/5583954/89336091-26f2ad00-d699-11ea-9f94-0a8da870eb0a.png)

**Expected behavior**
To look like in the example here: https://tiptap.dev/read-only

**Comment by hanspagel** [comment, marked as answer]
Depends on your code and your expected behaviour, but here is one possible solution:

Disable the outline for a read-only editor
```css
[contenteditable="false"]:focus {
  outline: none;
}
```

**Comment by janswist** [comment]
> Depends on your code and your expected behaviour, but here is one possible solution:
> 
> Disable the outline for a read-only editor
> 
> ```css
> [contenteditable="false"]:focus {
>   outline: none;
> }
> ```

Nice one! Works beautifully. Thanks!
